src/render-delaying-services.js

```diff
@@ -39,6 +39,7 @@ import {getServicePromise} from './service';
 const SERVICES = {
   'amp-dynamic-css-classes': '[custom-element=amp-dynamic-css-classes]',
   'variant': 'amp-experiment',
+  'amp-story-render': 'amp-story[standalone]',
```

Comment:
The validator team will need to update our list of render-blocking extensions so that this one is injected with a link-preload when necessary.

Reply:
/cc @gregable is that something you can do?  I'm not sure if it's already there; amp-story previously had a definition that was just broken.

Reply:
Internally we have b/77581738 filed which is about being able to remove the boilerplate for `amp-story` and not including a preload link. @newmuis Would the preload link be for all of amp-story (I don't see an amp-story-render-0.1.js being served from the cdn)?

Reply:
Right now, yes.  We could take it as a separate project to divide amp-story into what's render-blocking vs. what's not, but we're a while away from that.

Reply:
Filed b/129863998

Reply:
Annnnd, this is already being done. Apologies for the noise. If you're not seeing a preload please let me know.

Reply:
Thanks @honeybadgerdontcare!